### ISSUE TITLE
CplexSolver not defined

### ISSUE THREAD

**Issue by tdw1221** [JuliaOpt/CPLEX.jl#247]
Hi,
    I installed CPLEX on my mac and was able to successfully run Pkg.add("CPLEX") in julia. However, when trying to setup a model with solver=CplexSolver(), i got the error message saying: UndefVarError: CplexSolver not defined. Not sure what is causing this, I guess it may have something to do with the path of CPLEX?

thanks

**Comment by QunLin0** [comment]
maybe you can try the following:
m=Model(with_optimizer(CPLEX.Optimizer))

**Comment by odow** [comment]
`CplexSolver` has been deprecated. Use 
```Julia
using JuMP, CPLEX
model = Model(with_optimizer(CPLEX.Optimizer))
``` 
instead.

Closed by the v0.6.0 tag: https://github.com/JuliaOpt/CPLEX.jl/releases/tag/v0.6.0.
